poetry.lock

```diff
@@ -972,763 +1657,6 @@ docs = ["Sphinx", "sphinx_rtd_theme"]
 grpc = ["grpcio", "grpcio", "protobuf"]
 
 [metadata]
-lock-version = "1.1"
```

Comment:
I guess I shouldn't be surprised that Dependabot assumes we are using the latest Poetry version.

Reply:
I don't think changing that will help, based on the discussion in https://www.github.com/dependabot/dependabot-core/issues/1556